blocks/api/serializer.js

```diff
@@ -58,15 +57,6 @@ export function getSaveContent( blockType, attributes ) {
 		// Applying the filters adding extra props
 		const props = applyFilters( 'getSaveContent.extraProps', { ...element.props }, blockType, attributes );
 
-		// Adding the generated className
-		if ( !! className ) {
-			const updatedClassName = classnames(
-				className,
-				props.className,
-			);
-			props.className = updatedClassName;
-		}
-
 		return cloneElement( element, props );
 	};
 	const contentWithClassName = Children.map( saveContent, addExtraContainerProps );
```

Comment:
W might want to update this const to `contentWithExtraProps` to reflect changes introduced.